test/integration/steps/test_parent_steps.py

```diff
@@ -239,6 +207,25 @@ def change_event_with_break_in_opening_times(context):
 
 @given("the Changed Event has two breaks in opening times", target_fixture="context")
 def change_event_with_two_breaks_in_opening_times(context):
+    deletions = []
```

Comment:
What does line 211 to 215 do?
```
for count, times in enumerate(context["change_event"]["OpeningTimes"]):
        if times["Weekday"] == "Monday":
            deletions.insert(0, count)
    for entries in deletions:
        del context["change_event"]["OpeningTimes"][entries]
```

Reply:
We ran into a problem with opening times tickets where because we're building from ODS some had 1 monday entry, some had 2. 

If we're changing the opening times we need to make sure all Mondays are removed so that we can add new ones without them causing conflicts. The above code works as follows:

Loop through opening times in CE. Output index of each one to the front of an array. Loop through the array to delete indexes in the opening times from the last to the first. 

If we delete the index in the for loop, it causes problems with consecutive entries because it'll skip to the next.

This means that when we add new entries, we can guarantee there are no Monday entries in the Opening times to cause conflicts.